wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -1,60 +1,62 @@
 {% load wagtailadmin_tags i18n %}
 
-<div class="w-editing-sessions">
+<ol class="w-editing-sessions">
```

Comment:
Using `ol` as the list is technically ordered, even if it is not really transparent to the end user